install/kubernetes/cilium/templates/cilium-agent/serviceaccount.yaml

```diff
@@ -8,4 +8,13 @@ metadata:
   annotations:
     {{- toYaml .Values.serviceAccounts.cilium.annotations | nindent 4 }}
   {{- end }}
+  {{- if or .Values.serviceAccounts.cilium.annotations .Values.annotations }}
+  annotations:
+    {{- with .Values.annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+    {{- with .Values.serviceAccounts.cilium.annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- end }}
```

Comment:
@bradwhitfield This is causing a duplicate `annotations` key if `.Values.serviceAccounts.cilium.annotations` is specified in the Helm release. Would you mind sending in a PR to fix this?